nntrainer/layers/layer_node.cpp

```diff
@@ -586,20 +587,10 @@ void LayerNode::calcGradient() {
  * @brief Set the batch for the layer
  */
 void LayerNode::setBatch(unsigned int batch) {
-  /** @todo we won't going to need Layer::setBatch(InitLayerContext), remove it
-   */
-  if (hasInputShapeProperty()) {
-    auto &input_shapes =
-      std::get<std::vector<props::InputShape>>(*layer_node_props);
-    for (auto &input_shape : input_shapes) {
-      input_shape.get().batch(batch);
-    }
-  }
```

Comment:
@note: we don't (and should not) query input shape property for querying (current and actual) input dimension, by this we can segregate + detect bugs earlier which would happen by assuming input_shape has the batch dimension as well.